containers/images/core/container-assets/build/03_webpack

```diff
@@ -0,0 +1,28 @@
+#! /bin/bash -e
+
+# Enabling collections
+. /opt/rh/rh-ruby24/enable
+. /opt/rh/rh-nodejs8/enable
+
+# Download required npm packages
+npm i --no-optional
+
+# Asset compliations
+PLUGINS_PATHS=`./script/plugin_webpack_directories.rb | ruby -rjson -e "puts JSON.parse(STDIN.read)['paths'].join(' ')"`
+
+# run npm for each plugin and remove custom babel configuration
+for plugin in $PLUGINS_PATHS; do
+cd $plugin/..
+rm -f .babelrc
+npm i --no-optional
+cd -
```

Comment:
Indent these lines to show they belong to the for-loop?

Reply:
It's also simpler to use a subshell if you're going to use bash anyway:
```bash
for plugin in $PLUGIN_PATHS ; do
  (
    cd $plugin/..
    rm -f .babelrc
    npm i --no-optional
  )
done
```